src/app/lib/config/toggles/testConfig.js

```diff
@@ -47,4 +47,7 @@ export default {
   variantCookie: {
     enabled: true,
   },
+  storyPageAds: {
```

Comment:
The toggles in the files are also used as a default if the toggles request fails. Maybe you didn't mean remove them here, but instead just add to iSite, but just incase. :)

Reply:
This has been added to iSite. will leave the ones specified in this file for the reason @mmcculloch gave.

Reply:
> Can we move these toggles to iSite instead of having them within code in the app?

Yeah I'm a bit confused about the future of toggles in source control? Are we keeping the toggles as defaults if the request fails or are we getting rid of them and doing something else @chris-hinds ?

Reply:
It was my understanding that we were getting rid of local toggles

Reply:
If we are doing that, what happens if a request to the toggles endpoint fails?